src/server/zset_family.cc

```diff
@@ -84,7 +84,9 @@ struct ZParams {
 };
 
 OpResult<PrimeIterator> FindZEntry(const ZParams& zparams, const OpArgs& op_args, string_view key,
-                                   size_t member_len) {
+                                   size_t member_len, bool& created) {
```

Comment:
pass by pointer.

Reply:
Done, although the latest Google style-guide _does_ allow this :)